build/Prepare.targets

```diff
@@ -40,10 +40,16 @@
       <OverwriteExtractionDestination Condition="'$(OverwriteExtractionDestination)' == ''">True</OverwriteExtractionDestination>
     </PropertyGroup>
 
-    <DownloadFile Condition=" '@(_DownloadAndExtractItem)' != '' "
+
+    <DownloadFile Condition=" '@(_DownloadAndExtractItem)' != '' And '%(_DownloadAndExtractItem.Local)' != 'true'"
                   Uri="%(_DownloadAndExtractItem.Url)"
                   DestinationPath="%(_DownloadAndExtractItem.DownloadFileName)" />
 
+    <Copy Condition=" '@(_DownloadAndExtractItem)' != '' And '%(_DownloadAndExtractItem.Local)' == 'true'"
```

Comment:
When is local set?

Reply:
The idea is you set it in BundledToolset.props when you want to create a local build of core-sdk which consumes a local build of the toolset.